package.json

```diff
@@ -29,17 +29,17 @@
 		"dist/*"
 	],
 	"dependencies": {
-		"@enact/core": "^4.0.7",
-		"@enact/i18n": "^4.0.7",
-		"@enact/sandstone": "^2.0.2",
-		"@enact/spotlight": "^4.0.7",
-		"@enact/ui": "^4.0.7",
+		"@enact/core": "^4.5.0-alpha.1",
+		"@enact/i18n": "^4.5.0-alpha.1",
+		"@enact/sandstone": "^2.5.0-alpha.1",
+		"@enact/spotlight": "^4.5.0-alpha.1",
+		"@enact/ui": "^4.5.0-alpha.1",
 		"dotenv": "^6.2.0",
 		"node-fetch": "^2.3.0",
 		"prop-types": "^15.6.2",
 		"qs": "^6.6.0",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1",
+		"react": "^18.0.0",
+		"react-dom": "^18.0.0",
 		"react-router-dom": "^5.2.0",
 		"yargs": "^12.0.5"
 	},
```

Comment:
Does this repo use jest 24?

Reply:
It uses "jest": "^24.1.0" on line 48.

Reply:
I meant, shouldn't it update to at lest 27 since we are using it.

Reply:
Oh, yes. I'll update it as well. Thanks!